sklearn/preprocessing/imputation.py

```diff
@@ -375,3 +375,161 @@ def transform(self, X):
             X[coordinates] = values
 
         return X
+
+
+class MissingIndicator(BaseEstimator, TransformerMixin):
+    """Missing values indicator transformer for indicating missing values.
+
+    Parameters
+    ----------
+    missing_values : integer or "NaN", optional (default="NaN")
+        The placeholder for the missing values. All occurrences of
+        `missing_values` will be imputed. For missing values encoded as np.nan,
+        use the string value "NaN".
+
+    axis : integer, optional (default=0)
```

Comment:
I understand the missing indicator is independent of axis. So the meaning of ``missing_indicators =train`` would be only indicate the features with the missing values at fit time ?